src/Transforms/Repeater.php

```diff
@@ -0,0 +1,62 @@
+<?php namespace Leean\Acf\Transforms;
+
+/**
+ * Class Repeater.
+ *
+ * @package Leean\Acf\Transforms
+ */
+class Repeater
+{
+	/**
+	 * Apply the transforms
+	 *
+	 * @param array $field The field.
+	 * @return array
+	 */
+	public static function apply( $field ) {
```

Comment:
Yes, that's true. I think you should just have a static API and methods and objects behind or just plain inheritance.